src/main/java/de/medizininformatikinitiative/fhir_data_evaluator/populations/individuals/InitialIndividual.java

```diff
@@ -0,0 +1,20 @@
+package de.medizininformatikinitiative.fhir_data_evaluator.populations.individuals;
+
+import de.medizininformatikinitiative.fhir_data_evaluator.populations.InitialPopulation;
+
+/**
+ * Represents an individual of an initial population.
+ * <p>
+ * This does not hold any data because an initial population is always incremented by 1.
+ */
+public record InitialIndividual() implements Individual<InitialPopulation> {
```

Comment:
since this class holds no data, we don;t need a record. an interface with a single constant holding an anonymous instance would do the job